colibre/auto_plotter/halo_mass_functions.yml

```diff
@@ -18,3 +19,25 @@ halo_mass_function:
   observational_data:
     - filename: HaloMassFunction/Tinker2008.hdf5
     - filename: HaloMassFunction/Bocquet2016.hdf5
+
+halo_mass_function_adaptive:
+  type: "adaptivemassfunction"
+  select_structure_type: 10  
+  legend_loc: "lower left"
+  number_of_bins: 25
+  x:
+    quantity: "masses.mass_200crit"
+    units: Solar_Mass
+    start: 1e9
```

Comment:
Can you please use **the same** axis limits in `halo_mass_function_adaptive` plot and `halo_mass_function` plot?

Reply:
Good point.